core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/PrintInfo.java

```diff
@@ -39,10 +39,8 @@
   public static void printMetaBlockInfo(SiteConfiguration siteConfig, Configuration conf,
       FileSystem fs, Path path) throws IOException {
     FSDataInputStream fsin = fs.open(path);
-    BCFile.Reader bcfr = null;
-    try {
-      bcfr = new BCFile.Reader(fsin, fs.getFileStatus(path).getLen(), conf,
-          CryptoServiceFactory.newInstance(siteConfig, ClassloaderType.ACCUMULO));
+    try (BCFile.Reader bcfr = new BCFile.Reader(fsin, fs.getFileStatus(path).getLen(), conf,
```

Comment:
```suggestion
    try (var bcfr = new BCFile.Reader(fsin, fs.getFileStatus(path).getLen(), conf,
```

Reply:
@ctubbsii thanks for the info. I'm not that familiar with the use of var at the moment so I will read up on it a bit and then see if I can apply it where appropriate. You can take another look once I push another update.

Reply:
That's fine. We can merge this without those suggestions. They aren't necessary.